execution/executor-benchmark/src/pipeline.rs

```diff
@@ -144,7 +144,7 @@ where
             .name("txn_executor".to_string())
             .spawn(move || {
                 start_execution_rx.map(|rx| rx.recv());
-                let overall_measuring = OverallMeasuring::start();
+                let mut overall_measuring = OverallMeasuring::start();
```

Comment:
By making overall_measuring mutable, now in line 151 we will have 'stage_overall_measuring' which is another mutable cloned object from this object. Both the objects can completely diverge now. Maybe it is fine with the current logic, but this could also get complex in the future.

If it makes sense (a) we could remove event summary out of Overall Measuring or (b) do not clone stage_overall_measuring from overall_measuring